paymentsheet/src/main/java/com/stripe/android/paymentsheet/addresselement/AddressElementActivity.kt

```diff
@@ -57,22 +59,36 @@ internal class AddressElementActivity : ComponentActivity() {
         WindowCompat.setDecorFitsSystemWindows(window, false)
         starterArgs.config?.appearance?.parseAppearance()
 
+        starterArgs.statusBarColor?.let {
+            window.statusBarColor = it
+        }
+
         // set a default result in case the user closes the sheet manually
         setResult()
 
         setContent {
-            val modalBottomSheetState =
-                rememberModalBottomSheetState(
-                    ModalBottomSheetValue.Expanded
-                )
+            val modalBottomSheetState = rememberModalBottomSheetState(
+                initialValue = ModalBottomSheetValue.Hidden,
+                skipHalfExpanded = true,
+                confirmStateChange = {
+                    val route = navController.currentDestination?.route
+                    route != AddressElementScreen.Autocomplete.route
+                }
```

Comment:
I find that the user shouldn’t be able to exit from the sheet while in the autocomplete screen. Any concerns around this?